RoadRater/app/src/main/java/com/roadrater/presentation/components/ReviewCard.kt

```diff
@@ -56,6 +65,37 @@ fun ReviewCard(review: Review) {
 
             Spacer(modifier = Modifier.height(8.dp))
 
+            // Plate link
+            Surface(
+                modifier = Modifier
+                    .clip(RoundedCornerShape(8.dp))
+                    .clickable(enabled = onNumberPlateClick != null) {
```

Comment:
In what scenario is this null? (Also I did review to change it to empty instead of null) You are always passing in a value to I think its unnecessary
```suggestion
                    .clickable(onClick = onNumberPlateClick) {
```
and remove the curly bracket invoke